src/System.Memory/pkg/System.Memory.pkgproj

```diff
@@ -9,7 +12,6 @@
     <ProjectReference Include="..\src\System.Memory.csproj" />
     <InboxOnTargetFramework  Include="$(AllXamarinFrameworks)" />
     <InboxOnTargetFramework  Include="netcoreapp2.1" />
-    <InboxOnTargetFramework  Include="$(UAPvNextTFM)" />
```

Comment:
Do we think it is worth adding a condition around these? Otherwise when we do want to support these we will have to figure out all the necessary changes again.

Reply:
Will we ever do that out of this branch though? I suppose that when we eventually want to ship the next version of UAP's metapackage along with all of these packages, we will want to do the work to add the placeholders, but I'm not so sure if we need to do that on this branch.

Reply:
It isn't clear where they will ship yet, but if I were to guess I would say we should service these packages from this branch.

Reply:
ok, in that case I'll re-add these back with a condition.